docs/Dirb.md

```diff
@@ -0,0 +1,36 @@
+# How to install and use Dirb for Mac OS
+
+## Installation 🔧
+
+All you need to do is follow the commands below, and Dirb will be installed in no time!
+
+```sh
+curl -L https://sourceforge.net/projects/dirb/files/dirb/2.22/dirb222.tar.gz/download -o dirb222.tar.gz
+tar xvzf dirb222.tar.gz
+chmod 766 dirb222
+cd dirb222
+find . -type d -exec chmod 766 {} \;
+chmod +x configure
+./configure
+make && make install
```

Comment:
Suggestion: before compiling the code itself, we could encourage developers read the source code!

Reply:
It would be nice! I'll add it here!